versions/1/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -8,3 +8,7 @@ spec:
   properties:
     skip:
       $ref: "./../skip.spec.yml#/definitions/skip"
+    wait_for_data_timeout:
+      description: Short explanation for why test has been skipped.
+      type: string
```

Comment:
Q: Are types defined somewhere? Would it be possible to define a duration type for this strings?